packages/terra-list/src/ListItem.scss

```diff
@@ -1,30 +1,22 @@
 @import '~terra-mixins';
 @import './variables';
 
-.terra-ListItem-hasChevron {
-  display: block;
-  position: relative; // relative position so the chevron can use absolute position to vertically center itself
+.terra-ListItem {
+  display: table;
+  width: 100%;
+}
 
-  [dir='ltr'] & {
-    @include terra-padding-end(20px);
-  }
+.terra-ListItem > * {
+  display: table-cell;
+  width: 100%;
+}
 
-  [dir='rtl'] & {
-    @include terra-padding-start(20px);
-  }
+.terra-ListItem-chevron {
+  @include terra-padding-end(10px);
+  display: flex;
 
-  .terra-ListItem-chevron {
-    @include terra-position-end(10px);
-    color: $terra-list-chevron-color;
-    display: inline-block;
-    -webkit-font-smoothing: antialiased;
-    font-style: normal;
-    height: 20px;
-    position: absolute;
-    speak: none;
-    text-decoration: inherit;
-    top: 50%;
-    transform: translate(0, -50%);
+  > svg {
+    color: $terra-grey-30;
```

Comment:
What's the thoughts on changing this from using the `$terra-list-chevron-color` variable to `$terra-grey-30`? Is this something that should be themeable @neilpfeiffer?